static/styles/landing-page.scss

```diff
@@ -3356,6 +3356,7 @@ nav ul li.active::after {
 
     .portico-landing.integrations .integration-categories-dropdown {
         display: block;
+        width: 670px;
```

Comment:
I don't think hardcoding widths is the right approach here. I think it's better to have the dropdown expand to its maximum width while setting its margins to be of the same width as the lozenges instead.

Reply:
> margins to be of the same width as the lozenges
Yeah I kinda of invested a bit of time in doing that (ahh, more than a bit I guess). Thing I feel doing that is kinda same thing if we do width. See if we adjust margins, then we will need to adjust margins based on media queries for all the resolutions as well; there is no getting by that. 
But I kinda felt it was a bit more work to calculate exact margins because for different resolutions, the container of the lozenges changes size and then lozenges change size as well and therefore margin variation to be calculated becomes complex or a 2 step process.

With setting up widths we still need to do that for each resolution level (the media queries) but avoid the complex calculation of margins.

Thoughts @synicalsyntax ?

Reply:
![screenshot at jul 09 14-06-52](https://user-images.githubusercontent.com/15116870/42476245-2db35098-8382-11e8-8861-06a5c230aeee.png)

As seen above, the container of the dropdown for some reason is also highlighted when you select it, and it looks like it can be removed if you set padding for `.dropdown-toggle` to be 0 pixels. 

However, this makes `.dropdown-category-label` fill up the entire space, and specifying its width doesn't center it in the middle, while specifying its margins will allow it to be centered properly.

Reply:
So if we do that problem is with the fact that the lozenges have constant widths between two media query levels and rather have their margins shrink and expand dynamically than changing widths so that they remain at centre.
Gif below for the margins code I think you are talking about (commit diff at the end):
![integrations](https://user-images.githubusercontent.com/17237412/42502961-01da4b60-8455-11e8-9ac8-a5e27c44dc89.gif)

Commit diff:
```diff
@ -2585,6 +2585,7 @@ nav ul li.active::after {
     cursor: pointer;
     margin: 30px 0;
     display: none;
+    padding: 0px;
 }
 
 .portico-landing.integrations .integration-lozenges {
@@ -3368,19 +3369,21 @@ nav ul li.active::after {
 
     .portico-landing.integrations .integration-categories-dropdown .dropdown-toggle {
         position: relative;
-        padding: 0 25px;
+        margin-left: 89px;
+        margin-right: 89px;
     }
 
     .portico-landing.integrations .integration-categories-dropdown i {
         position: absolute;
         top: 10px;
-        right: 36px;
+        right: 15px;
         font-size: 24px;
     }
 
     .portico-landing.integrations .integration-categories-dropdown .dropdown-list {
         display: none;
-        padding: 0 25px;
+        margin-left: 89px;
+        margin-right: 89px;
     }
 
     .portico-landing.integrations .integration-categories-dropdown h3 {
```

Reply:
My main issue is with the dropdown focus bug I pointed out above, so if you believe it's possible to eliminate that bug while using screen-width based width attributes, I'd still be fine with that approach.